python/ray/workflow/api.py

```diff
@@ -434,106 +378,92 @@ def delete(workflow_id: str) -> None:
     wf_storage.delete_workflow()
 
 
-WaitResult = Tuple[List[Any], List[Workflow]]
+T = TypeVar("T")
 
 
 @PublicAPI(stability="beta")
-def wait(
```

Comment:
Should we also remove the wait from the doc for now?

Reply:
removed